src/R/send-charts.R

```diff
@@ -362,6 +362,33 @@ for (s2 in settings) {
 sankey(df_joiners_trans, "Joiner Transitions")
 ggsave("../../target/Joiner_Transitions.pdf")
 
+
+### Ribbon plot ###
+
+ribbon <- function(data, title) {
+  ggplot(data, aes(year, upper0)) +
+    geom_ribbon(aes(ymax = upper0, ymin = lower0, fill = "#1b9e77"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper1, ymin = lower1, fill = "#d95f02"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper2, ymin = lower2, fill = "#7570b3"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper3, ymin = lower3, fill = "#e7298a"), alpha = 0.2) +
+    ggtitle(paste(title, "probability by academic year")) +
+    xlab("NCY") +
+    ylab("95% probability interval") +
+    scale_fill_manual(name = "Years", labels = c("2013", "2014", "2015", "2016"), values = c("#1b9e77", "#d95f02", "#7570b3", "#e7298a")) +
```

Comment:
I think we probably need to infer this automatically since e.g. the Camden data only has three years of transitions.